src/ContentUtils.elm

```diff
@@ -24,6 +25,19 @@ filterByContentType contentType content =
     List.filter (\c -> c.contentType == contentType) content
 
 
+filterByTitle : String -> List Content
+filterByTitle title =
+    let
+        filterdList =
+            List.filter (\c -> contains (toLower title) (toLower c.title))
+                Posts.posts
+    in
+        if (List.isEmpty filterdList) then
+            postsInOrder
+        else
+            filterdList
```

Comment:
You've got a typo here - `filterdList` needs to be `filteredList`.

I'd probably pull out the function you pass to `List.filter` out, and then this code can be a bit shorter.

(It would need to be updated a bit if you model the filter as a `Maybe String` too)